pkg/plugin/response/response.go

```
@@ -0,0 +1,181 @@
+
+// CheckAndRespWithData 判断是否出现错误，并返回对应响应（带data数据）
+func CheckAndRespWithData(c *gin.Context, data interface{}, err error) {
+	if IsFailWithResp(c, err) {
+		return
+	}
+	OkWithData(c, data)
+}
+
+// CheckErr 校验未知错误并抛出
+func CheckErr(err error, template string, args ...interface{}) (e error) {
+	args = append(args, err)
```

golangci-lint (1.64.8)

ineffectual assignment to args

(ineffassign)